tribits/common_tpls/FindTPLHDF5.cmake

```diff
@@ -37,6 +37,13 @@ IF (HDF5_ALLOW_PREFIND)
 
   FIND_PACKAGE(HDF5 COMPONENTS ${HDF5_COMPNENTS})
 
+  # Make sure that HDF5 is parallel.
```

Comment:
Nico,

Don't we need to check to see if the project is being built in MPI mode or not first?  If the project is being built in SERIAL mode, then is not a serial version of HDF5 not just fine?